### DISCUSSION TITLE
app start first time

### DISCUSSION THREAD

**Discussion by ggslayer** [tauri-apps/tauri#8558] [Q&A] [answered]
Hi, I want enable autostart when the app first start in user's computer after install.
Is there availeble data identity the first time which I can use?   if no, I will write config info myself, it's easy also, but I want to know the availeble one if exists..

**Comment by FabianLars** [comment, marked as answer]
No, there is no existing config yet.

**Reply by ggslayer** [reply]
ok

**Comment by arashi-dev** [comment]
Maybe this does help?
https://github.com/tauri-apps/tauri/issues/2589#issuecomment-1297341483
